addons/l10n_ar_withholding/models/l10n_ar_earnings_scale.py

```diff
@@ -0,0 +1,47 @@
+from odoo import models, fields
+
+
+class L10nArEarningsScale(models.Model):
+    _name = 'l10n_ar.earnings.scale'
+    _description = 'l10n_ar.earnings.scale'
+
+    name = fields.Char(required=True)
+    line_ids = fields.One2many('l10n_ar.earnings.scale.detail', 'scale_id')
+
+
+class L10nArEarningsScaleDetail(models.Model):
+    _name = 'l10n_ar.earnings.scale.detail'
+    _description = 'l10n_ar.earnings.scale.detail'
+    _order = 'from_amount, to_amount'
+
+    scale_id = fields.Many2one(
+        'l10n_ar.earnings.scale', required=True, ondelete='cascade',
+        help="Calculation of the withholding amount: From the taxable amount (tax base + tax bases applied this month to same tax and partner - non-taxable minimum) subtract the immediately previous amount of the column 'S/ Exced. de $' to detect which row to work with and apply the percentage of said row to the result of the subtraction. Then add to this amount the amount of the '$' column."
+    )
+    currency_id = fields.Many2one(
+        'res.currency', default=lambda self: self.env.ref('base.ARS'), store=False
+    )
+    from_amount = fields.Monetary(
```

Comment:
Should not we make from amount  a compute?

Reply:
I think no because from_amount is a fixed amount that we set on addons/l10n_ar_withholding/data/earnings_table_data.xml . @vbe-odoo

Reply:
@pablohmontenegro but can't we use the same column (`to_amount`), as for the 1st row it will start from 0, and from there, the from amount will be the previous value in the `to_amount` column

Reply:
And remove the `from_amount` from the data.